packages/app/src/components/applicationStages/ApplicationStages.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import './style.css';
+
+const ApplicationStages = () => {
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [currentStatus, setCurrentStatus] = useState('received');
+  const status = [
+    'Received',
+    'Awaiting HIPAA',
+    'HIPAA Verified',
+    'Document Review',
+    'Ready for Board Review',
+  ];
+
+  useEffect(() => {
+    setCurrentStatus(activeNomination.status);
+  }, [currentStatus]);
+
+  function advanceStage(activeNomStatus) {
+    switch (activeNomStatus) {
```

Comment:
This is a pretty big switch statement you have here, and would need to be altered if we ever add or change statuses down the road.

Can you think of a more automated way of advancing between the statuses?

Hint: Have you considered storing a reference to the current status in your component's state, rather that storing the current status itself?

Reply:
changed to method on index